src/frontend/apps/desk/src/features/teams/api/useTeam.tsx

```diff
@@ -0,0 +1,35 @@
+import { UseQueryOptions, useQuery } from '@tanstack/react-query';
+
+import { fetchAPI } from '@/api';
+
+import { TeamResponse } from './types';
+
+export type TeamParams = {
+  id: string;
+};
+
+export interface TeamResponseError {
+  detail: string;
+}
+
+export const getTeam = async ({ id }: TeamParams): Promise<TeamResponse> => {
+  const response = await fetchAPI(`teams/${id}`);
+
+  if (!response.ok) {
+    throw new Error(`Couldn't fetch team: ${response.statusText}`);
+  }
+  return response.json() as Promise<TeamResponse>;
+};
+
+export const KEY_LIST_TEAM = 'team';
```

Comment:
in `api/useTeams`, this constant has the same value, which is a bit misleading. Wdyt updating `KEY_LIST_TEAM` to `KEY_LIST_TEAMS` [here](https://github.com/numerique-gouv/people/pull/51/files#diff-3b1b170393fb329ba150019552763720f6472c957998464a8d9fedcb2779af24L55)

Reply:
nit: `KEY_LIST_...` is a bad naming, as the same naming key is used for `get`, `retrieve`, `create` operations